### ISSUE TITLE
Run with an error "double free or corruption"

### ISSUE THREAD

**Issue by nmaac** [tobegit3hub/tensorflow_template_application#20]
I ran the sparse_classifier.py program and got this error in the **export_model()** function.

INFO:tensorflow:./sparse_model/00000001-tmp/export is not in all_model_checkpoint_pat
hs. Manually adding it.
*** Error in `python': double free or corruption (!prev): 0x0000000000825660 ***

I want to test the AUC of the entire validation set, setting the validation_batch as the size of validation set is too slow. So I want to export the model and predict AUC on the entire validation set, and I got this problem. 

Is there any suggestion？

**Comment by tobegit3hub** [comment]
Hi @manene .

Of course you can export the model and predict the AUC for all the validation set.

I don't have the corruption problem and successfully export the model with the latest code. What's the version of TensorFlow are you using?

**Comment by nmaac** [comment]
Hi @tobegit3hub 
I use TensorFlow 1.0 and Python 2.7

**Comment by manene** [comment]
detail error log:

*** Error in `python': double free or corruption (!prev): 0x0000000000825610 ***
======= Backtrace: =========
/lib64/libc.so.6(+0x78466)[0x7f850660e466]
/lib64/libc.so.6(+0x7a3b3)[0x7f85066103b3]
/lib64/ld-linux-x86-64.so.2(_dl_deallocate_tls+0x67)[0x7f8507636797]
/lib64/libpthread.so.0(+0x6add)[0x7f8507050add]
/lib64/libpthread.so.0(+0x6bec)[0x7f8507050bec]
/lib64/libpthread.so.0(pthread_join+0xd4)[0x7f85070520c4]
/.../opt/gcc46/lib64/libstdc++.so.6(_ZNSt6thread4joinEv+0x27)[0x7f84dbd0
d0c7]
/.../lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow.so
(+0x23b3af0)[0x7f84d2281af0]
/.../lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow.so
(_ZN10tensorflow6thread10ThreadPool4ImplD0Ev+0xb3)[0x7f84d225e693]
/.../lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow.so
(_ZN10tensorflow6thread10ThreadPoolD1Ev+0x1a)[0x7f84d225ecfa]
/.../lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow.so
(_ZN10tensorflow10FileSystem16GetMatchingPathsERKSsPSt6vectorISsSaISsEE+0x599)[0x7f84
d227e729]

**Comment by tobegit3hub** [comment]
I don't need this problem in server environments.

Have you fixed this? @manene
